assets/icons.json

```diff
@@ -312,6 +312,12 @@
     ],
     "viewBox": "0 0 24 24"
   },
+  "update": {
+    "path": [
+      "M483 936q-75 0-141-28.5T226.5 830q-49.5-49-78-115T120 574q0-75 28.5-140t78-113.5Q276 272 342 244t141-28q80 0 151.5 35T758 347V241h60v208H609v-60h105q-44-51-103.5-82T483 276q-125 0-214 85.5T180 571q0 127 88 216t215 89q125 0 211-88t86-213h60q0 150-104 255.5T483 936Zm122-197L451 587V373h60v189l137 134-43 43Z"
+    ],
+    "viewBox": "0 96 960 960"
```

Comment:
Is there a reason the viewBox of the svg looks different that the other ones in the file `0 0 24 24`?

Reply:
No reason - we got the file and it was like this. We can reduce it